gateway/handlers/reader.go

```diff
@@ -22,13 +23,22 @@ import (
 func MakeFunctionReader(metricsOptions metrics.MetricOptions, c client.ServiceAPIClient) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 
+		verbose := false
+
+		if r.URL != nil {
+			verboseMode := r.URL.Query().Get("v")
+			if len(verboseMode) != 0 {
```

Comment:
May want to check actual accepted values here (i.e. for the `?v=0` or `?v=false` case.

The Docker API uses a utility for this - definitely not perfect as well; I think I would've preferred a more "strict" approach, but it does the job for most values that people would "expect" to work; https://github.com/moby/moby/blob/master/api/server/httputils/form.go#L9-L22

```
// BoolValue transforms a form value in different formats into a boolean type.
func BoolValue(r *http.Request, k string) bool {
	s := strings.ToLower(strings.TrimSpace(r.FormValue(k)))
	return !(s == "" || s == "0" || s == "no" || s == "false" || s == "none")
}

// BoolValueOrDefault returns the default bool passed if the query param is
// missing, otherwise it's just a proxy to boolValue above.
func BoolValueOrDefault(r *http.Request, k string, d bool) bool {
	if _, ok := r.Form[k]; !ok {
		return d
	}
	return BoolValue(r, k)
}
```

Reply:
Didn't pick up on this in the review, good point. Also should this be verbose at the API level (it makes sense at the CLI level) maybe here it should be `inspect` or similar?